products/BellHybrid/services/db/agents/ShuffleQuoteModel.hpp

```diff
@@ -12,6 +12,7 @@ namespace Quotes
 {
     inline constexpr auto customGroup     = "Custom";
     inline constexpr auto predefinedGroup = "Predefined";
+    inline constexpr auto atMidnight      = "at midnight";
```

Comment:
I'd avoid using whitespace characters in such strings, what about `AtMidnight`?

Reply:
done

Reply:
Please update it also in `up.sql`

Reply:
done